src/interfaces/wallet.h

```diff
@@ -88,7 +88,7 @@ class Wallet
     virtual std::string getWalletName() = 0;
 
     // Get a new address.
-    virtual util::Result<CTxDestination> getNewDestination(const OutputType type, const std::string label) = 0;
+    virtual util::Result<CTxDestination> getNewDestination(const OutputType type, const std::string& label) = 0;
```

Comment:
Thanks for picking up https://github.com/bitcoin/bitcoin/pull/25721#discussion_r938747475 here and fixing up indentation in this file.

Reply:
The wallet method is still creates a copy?

```
$ git grep GetNewDestination -- '*wallet.h'
src/wallet/wallet.h:    util::Result<CTxDestination> GetNewDestination(const OutputType type, const std::string label);